blockwork/config/config.py

```diff
@@ -1,6 +1,90 @@
+from pathlib import Path
+from typing import Iterable, cast, overload, TYPE_CHECKING
 from blockwork.common.registry import RegisteredClass
```

Comment:
Why is this using absolute imports?

Reply:
Fixed this instance, will fix others in follow-up